configure.py

```diff
@@ -579,7 +579,7 @@
 parser.add_option('--enable-asan',
     action='store_true',
     dest='enable_asan',
-    help='build with asan')
+    help='build with address sanitization')
```

Comment:
Reading this again, I'm not sure _sanitization_ is the right word to describe what ASAN is. What if we use V8's definition:

```suggestion
    help='compile for Address Sanitizer to find memory bugs')
```